src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+          });
+    }
+    return t;
+  }
+  e = e && Object.prototype.hasOwnProperty.call(e, "default") ? e.default : e;
+  function a(t) {
+    var n = this,
+      i = !1;
+    return (
+      e(this).one(l.TRANSITION_END, function () {
+        i = !0;
+      }),
+      setTimeout(function () {
+        i || l.triggerTransitionEnd(n);
+      }, t),
+      this
+    );
+  }
+  var l = {
+    TRANSITION_END: "bsTransitionEnd",
+    getUID: function (t) {
+      do {
+        t += ~~(1e6 * Math.random());
+      } while (document.getElementById(t));
+      return t;
+    },
+    getSelectorFromElement: function (t) {
+      var e = t.getAttribute("data-target");
+      if (!e || "#" === e) {
+        var n = t.getAttribute("href");
+        e = n && "#" !== n ? n.trim() : "";
+      }
+      try {
+        return document.querySelector(e) ? e : null;
+      } catch (t) {
+        return null;
+      }
+    },
+    getTransitionDurationFromElement: function (t) {
+      if (!t) return 0;
+      var n = e(t).css("transition-duration"),
+        i = e(t).css("transition-delay"),
+        o = parseFloat(n),
+        r = parseFloat(i);
+      return o || r
+        ? ((n = n.split(",")[0]),
+          (i = i.split(",")[0]),
+          1e3 * (parseFloat(n) + parseFloat(i)))
+        : 0;
+    },
+    reflow: function (t) {
+      return t.offsetHeight;
+    },
+    triggerTransitionEnd: function (t) {
+      e(t).trigger("transitionend");
+    },
+    supportsTransitionEnd: function () {
+      return Boolean("transitionend");
+    },
+    isElement: function (t) {
+      return (t[0] || t).nodeType;
+    },
+    typeCheckConfig: function (t, e, n) {
+      for (var i in n)
+        if (Object.prototype.hasOwnProperty.call(n, i)) {
+          var o = n[i],
+            r = e[i],
+            s =
+              r && l.isElement(r)
+                ? "element"
+                : null === (a = r) || "undefined" == typeof a
+                ? "" + a
+                : {}.toString
+                    .call(a)
+                    .match(/\s([a-z]+)/i)[1]
+                    .toLowerCase();
+          if (!new RegExp(o).test(s))
+            throw new Error(
+              t.toUpperCase() +
+                ': Option "' +
+                i +
+                '" provided type "' +
+                s +
+                '" but expected type "' +
+                o +
+                '".'
+            );
+        }
+      var a;
+    },
+    findShadowRoot: function (t) {
+      if (!document.documentElement.attachShadow) return null;
+      if ("function" == typeof t.getRootNode) {
+        var e = t.getRootNode();
+        return e instanceof ShadowRoot ? e : null;
+      }
+      return t instanceof ShadowRoot
+        ? t
+        : t.parentNode
+        ? l.findShadowRoot(t.parentNode)
+        : null;
+    },
+    jQueryDetection: function () {
+      if ("undefined" == typeof e)
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)